jruby/neo4j/driver/ext/ruby_converter.rb

```diff
@@ -22,13 +22,26 @@ def as_ruby_object
             Neo4j::Driver::Point.new(srid: point.srid, x: point.x, y: point.y, z: nullable(point.z))
           when Java::OrgNeo4jDriverInternalTypes::TypeConstructor::BYTES
             Neo4j::Driver::ByteArray.new(String.from_java_bytes(as_byte_array))
+          when Java::OrgNeo4jDriverInternalTypes::TypeConstructor::DATE_TIME
+            to_time
           else
             as_object
           end
         end
 
         private
 
+        def to_time
+          time = as_zoned_date_time
```

Comment:
@klobuczek in general looks good but where is this method from

Reply:
https://neo4j.com/docs/api/java-driver/current/org/neo4j/driver/v1/Value.html#asZonedDateTime--

Reply:
I keep forgetting about that magical feature that everything falls back to java